.editorconfig

```diff
@@ -7,7 +7,6 @@ root = true
 charset = utf-8
 
```

Comment:
Lets add a way to handle the line-ending issue I was discussing on discord

Reply:
@jwu910 we didn't add line ending solution hahah. @psyduc can you add "LF" for the line ending, `end_of_line = lf`

Reply:
alright done